libraries/matrix/src/main/sqldelight/io/element/android/libraries/matrix/session/SessionData.sq

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE SessionData (
```

Comment:
Same remarks than for the SessionStore impl, we should move that to a module as it doesn't belong to the matrix module to know that.

Reply:
Should I just create a new 'elementdb' library module containing this? Or maybe a `storage` feature module?